src/app/projects/create-project/create-project.styl

```diff
@@ -87,6 +87,24 @@ md-select.create-project-select
   text-align center
   line-height 40px
 
+.create-project-panel-progress-item-number-in-progress
+  animation: pulse 5s infinite;
+
+@keyframes pulse {
+  0% {
+    background-color white
+    color black
+  }
+  50% {
+    background-color $item-enabled-color-bg
+    color white
+  }
+  100% {
+    background-color white
+    color black
+  }
+}
+
```

Comment:
Great if so! It was not working when I did animation for CodeNow button, so I decided to pay attention on that.